go.mod

```diff
@@ -3,53 +3,53 @@ module volcano.sh/volcano
 go 1.14
 
 require (
-	github.com/fsnotify/fsnotify v1.4.7
+	github.com/fsnotify/fsnotify v1.4.9
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
 	github.com/hashicorp/go-multierror v1.0.0
 	github.com/onsi/ginkgo v1.11.0
 	github.com/onsi/gomega v1.7.0
-	github.com/prometheus/client_golang v1.0.0
-	github.com/spf13/cobra v0.0.5
+	github.com/prometheus/client_golang v1.7.1
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
-	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
+	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
+	golang.org/x/time v0.0.0-20191024005414-555d28b269f0
 	gopkg.in/yaml.v2 v2.2.8
-	k8s.io/api v0.18.3
-	k8s.io/apimachinery v0.18.3
-	k8s.io/client-go v0.18.3
-	k8s.io/component-base v0.18.3
+	k8s.io/api v0.19.6
+	k8s.io/apimachinery v0.19.6
+	k8s.io/client-go v0.19.6
+	k8s.io/component-base v0.19.6
 	k8s.io/klog v1.0.0
 	k8s.io/kube-scheduler v0.0.0
-	k8s.io/kubernetes v1.18.3
+	k8s.io/kubernetes v1.19.6
 	sigs.k8s.io/yaml v1.2.0
-	volcano.sh/apis v1.3.0-k8s1.18.3-alpha.3
+	volcano.sh/apis v1.2.0-k8s1.19.6-alpha.1
```

Comment:
Volcano.sh/volcano should require the tag-commit from volcano/api master branch, other than release-1.19 branch.

Reply:
Let me release a normal version